RNTester/Podfile.lock

```diff
@@ -501,32 +521,35 @@ SPEC CHECKSUMS:
-  React-CoreModules: 86a1e291a562989319e44c2f7a9be64dfb7dd165
-  React-cxxreact: fd69a2b7c7af6f2f6918503430f004a6cd1abce7
-  React-jsi: c98bf484499763815354b02d80d861cdc3f4a64f
-  React-jsiexecutor: 0d0a1c3dbbf9b5bd64387aa90dc3b27bab93c0de
-  React-jsinspector: ad2f99f512a8e22e48d037dd04fb686710c85a1f
-  React-RCTActionSheet: 3c2cafdfaff98e7cb40e0cd5b3c58981b6098ef8
-  React-RCTAnimation: 52cbc676c289a54e6d53ff2b307bd1494a762755
-  React-RCTBlob: 563d8cadd1493a1659475bcd67681dfb64555563
-  React-RCTImage: b1f35b220756b2cf8e35e870afb6985b734277c7
-  React-RCTLinking: b43784e99974ecb338ec9c930c39e3d99dbe6c50
-  React-RCTNetwork: c1b88002c4f8534158010be237329d15c7b8c116
-  React-RCTPushNotification: d47f806e0a6e5047e3a7608e967f871461bcf39f
-  React-RCTSettings: e8d8db76845188b9a7ca1fa63a56260a6d245cac
-  React-RCTTest: 01bf6c6ac33584453fafb21ae421bfbc39f53714
-  React-RCTText: b90266a16b6988709a369e10c8b27e1872c79add
-  React-RCTVibration: 75d0ae67165d904a7b34cfec65f9c7ae5fdaf364
-  ReactCommon: 8850a0f5e4fbaef4b6b5c0fea0cf5c439d1ac4b2
-  Yoga: 5c142ea67c974bf340e36c57851d11c128ea23de
+  RCTRequired: b1425a4d370343402bed882d1661a809326ac43c
+  RCTTypeSafety: 49f6857826c4d3611b9fb2cbcea8ca2d943c88b1
+  React: bc17a68f02bbab66ce8948d8c56329eb27e21f61
+  React-ART: 46ecddf64c1ea973a7b63d74168a36a4113b60bd
+  React-callinvoker: 6a28d9affc37e63de1fd056226a7e13ffd1594d0
+  React-Core: ce714ac2b301d49e81a09c7645b580104a085a92
+  React-CoreModules: ca31277a950263e164ab97fb9774bdcfa0443b5c
+  React-cxxreact: 2068efcff9cff57daa5a060146fa77597e5f6b38
+  React-jsi: e1b8b0ce9da53a04e97c528ecc211bf8061533bf
+  React-jsiexecutor: dea36428055f1cd14b0ea971dbdd6c910578c948
+  React-jsinspector: a2828f7a110a3035847073b04604500fd5f6be12
+  React-RCTActionSheet: 8d4143e0b749c9d867c97e9d8bc16ca576ea576e
+  React-RCTAnimation: f123975b75c5fd64c9daf3f57fc838b26010bf16
+  React-RCTBlob: 810689af8a73abc31a91601f49a38ccc61e1fa62
+  React-RCTImage: 34161df0410d8fff44691c9cb55a3ef642132390
+  React-RCTLinking: 11cf686d5073c3fe2c1cc2b5a9873f8feea8e3d5
+  React-RCTNetwork: 48a715f368f5fa97291543a6b32fc0e5a4d843f8
+  React-RCTPushNotification: c6d457a7784edd2ed62f6a998ce90cb00d105b40
+  React-RCTSettings: 512f58e0db07388917825b796b29aab071816a57
+  React-RCTTest: 806969b2f7bb372ead51339dc53b1f7ab79881e0
+  React-RCTText: 478a98001793276d1c7b8b034000a2ee05fd9c7a
+  React-RCTVibration: 12e71074f01816c29caf9d8a5fb5dc98934ee390
+  React-TurboModuleCxx-RNW: abf7679d371bc25212a2dd91864c5c3e3e043267
+  React-TurboModuleCxx-WinRTPort: 1d75104033ba6e539ed1e3111bd6e35e7ce3b830
+  React-TurboModuleCxx-WinRTShared: b071c053b74a94a204ea2dc87b622795aebd067c
+  ReactCommon: 0b2b124b9b91c9cd5bd7538ba265538951e46fbf
+  Yoga: e79d814734797bc6ff12b2b7702224983a425047
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: f92cb3c61bb71bceed9b9afe3944fbd8451be8b9
+PODFILE CHECKSUM: 453d98c1a4e763cdbf636ce3570660198629bdf1
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.3
```

Comment:
Can you either install CocoaPods v1.8.4 system-wide `gem install cocoapods --version '= 1.8.4'` _or_ use the provided `Gemfile` to use the correct version?

```
gem install bundler
bundle install
bundle exec pod install
```

Reply:
fixed